blog/6-streampark-usercase-joyme.md

```diff
@@ -69,93 +69,93 @@ CREATE TABLE sink_table (
 'connector.write.max-retries' = '3'
 );
 
--- 代码逻辑过
+-- Code logic pass
 INSERT INTO sink_table
 SELECT  Data.uid  FROM source_table;
 ```
 
-### **2. 添加依赖**
+### **2. Dependency Addition**
 
-关于依赖这块是 StreamPark 里特有的，在 StreamPark 中创新型的将一个完整的 Flink Sql 任务拆分成两部分组成: Sql 和 依赖, Sql 很好理解不多啰嗦, 依赖是 Sql 里需要用到的一些 Connector 的 Jar, 如 Sql 里用到了 Kafka 和 MySQL 的 Connector, 那就需要引入这两个 Connector 的依赖, 在 StreamPark 中添加依赖两种方式，一种是基于标准的 Maven pom 坐标方式，另一种是从本地上传需要的 Jar 。这两种也可以混着用，按需添加，点击应用即可， 在提交作业的时候就会自动加载这些依赖。
+In terms of dependencies, StreamPark has a unique approach. A complete Flink SQL job is innovatively split into two components within StreamPark: the SQL and the dependencies. The SQL part is self-explanatory and requires no further explanation, but the dependencies are the various Connector JARs needed within the SQL, such as Kafka and MySQL Connectors. If these are used within the SQL, then these Connector dependencies must be introduced. In StreamPark, there are two ways to add dependencies: one is based on the standard Maven pom coordinates, and the other is by uploading the required Jars from a local source. These two methods can also be mixed and used as needed; simply apply, and these dependencies will be automatically loaded when the job is submitted.
```

Comment:
In terms of dependencies, it's an unique feature to Streampark.
The SQL part is easy to understand and requires no further explanation